### DISCUSSION TITLE
[&hide_title=true] affected "Total Commits" results

### DISCUSSION THREAD

**Discussion by szh-bash** [anuraghazra/github-readme-stats#2371] [Q&A] [answered]
### Describe the bug

with &hide_title=true, commits = 1.7k
without &hide_title=true, commits = 348

check [here](https://github.com/szh-bash/szh-bash/edit/main/README.md)

### Expected behaviour

same commits

**Comment by rickstaa** [comment, marked as answer]
@szh-bash Thanks for your report. It should and can not affect the total commits:

**Without hide_title=true**

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=szh-bash&adfsafsaf=asdfsafdsfa)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=szh-bash&adfsafsaf=asdfsafdsfa)](https://github.com/anuraghazra/github-readme-stats)
```

**With hide_title=true**

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=szh-bash&hide_title=true&adfsafsaf=asdfsafdsfa)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=szh-bash&hide_title=true&adfsafsaf=asdfsafdsfa)](https://github.com/anuraghazra/github-readme-stats)
```

It was caused by the GraphQL API returning an incorrect value, and then that value was cached. I added `&adfsafsaf=asdfsafdsfa` to break the cache (see https://github.com/anuraghazra/github-readme-stats#common-options).

**Reply by szh-bash** [reply]
However, i have 1.3k+ commits in 2022 and around 60+ commits in 2023.
Check your reply please, those are still wrong results...
![image](https://user-images.githubusercontent.com/27895812/211127021-c0c6d235-dcdb-4ac5-8c6b-d116a1db0051.png)

**Reply by Zo-Bro-23** [reply]
@szh-bash please see #2501. The `Total Commits (2023)` title is slightly misleading because it shows the total commits in the past 365 days, and not in 2023 alone. That is, you have 1.3k commits in the past 365 days. We are working to fix this issue!

**Comment by KatarzynaChmielecka** [comment]
I'm not sure what exactly author describe ;) But Total Commits for a year isn't correct. it still counts as 2022 :)

**Reply by szh-bash** [reply]
I think we need add "&random_string=random_string" in api to break cache.
Yep but Total Commits is still wrong :(

**Reply by rickstaa** [reply]
I think you are interested in https://github.com/anuraghazra/github-readme-stats/pull/2107/files. I will look at this PR later this month.
